arrows/core/derive_metadata.cxx

```diff
@@ -233,20 +233,23 @@ double
 compute_horizontal_gsd( double slant_range, double sensor_horizontal_fov,
                         double frame_width )
 {
-  return 2.0 * slant_range *
-         tan( ( sensor_horizontal_fov * kv::deg_to_rad ) / 2.0 ) / frame_width;
```

Comment:
Ack

Reply:
It probably would have been best to make bug fixes for calculations in a separate branch from refactoring yaw-pitch-roll utility functions.  This would have allowed the simple fixes to land right away without getting held up by more complex changes that require more review.  However, no need to refactor PRs at this point.